README.md

```diff
@@ -40,84 +39,160 @@ Contact mikeb@cc.gatech.edu for more information on the Aggie project.
 1. Navigate to `https://localhost:3000` in your browser.
     - This will show you the running site. Login with the user name and password from your terminal mentioned above.
     - If you did not set up the SSL certificate, use `http://localhost:3000` instead
 
+### Semi-automated installation script
+
+This is intended for setup on a fresh Ubuntu v18.04 system. Setup may need to be modified for other linux systems.
+
+If it says "user input", you won't want to paste anything beyond that until addressing the input.
+
+```shell script
+# Set up system
+
+export EDITOR=vim # Option 1
+export EDITOR=nano # Option 2
+
+sudo apt update
+sudo apt install -y ntp nginx software-properties-common
+sudo systemctl enable ntp
+sudo add-apt-repository universe
+sudo add-apt-repository ppa:certbot/certbot
+sudo apt update
+sudo apt install -y certbot python3-certbot-nginx
+
+sudo curl -o /etc/nginx/sites-available/aggie.conf https://raw.githubusercontent.com/TID-Lab/aggie/develop/docs/content/aggie-nginx
+sudo ln -s /etc/nginx/sites-available/aggie.conf /etc/nginx/sites-enabled/
+sudo rm /etc/nginx/sites-enabled/default
+sudo systemctl restart nginx
+# User input: Customize nginx settings with your domain name.
+$EDITOR /etc/nginx/sites-available/aggie.conf
+
+# User input: Set up SSL with a couple of prompts.
+sudo certbot --nginx
+
+# User input: Set up SSL certificate auto-renewal.
+crontab -e
+# Paste the following line in crontab, replacing `X` with the current minutes + 1
+# (e.g. if it's 12:15pm, write `16` instead of `X`):
+#   X * * * * PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin && sudo /usr/bin/certbot renew --no-self-upgrade > ${HOME}/certbot-cron.log 2>&1
+#
+# Then wait a minute and verify that it attempted a renewal: 
+cat ~/certbot-cron.log
+
+# Mongo DB. Source: https://docs.mongodb.com/v4.2/tutorial/install-mongodb-on-ubuntu/
+wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+sudo apt update
+sudo apt install -y mongodb-org
+sudo systemctl enable mongod
+# Optional: Increase ulimits via https://docs.mongodb.com/manual/reference/ulimit/.
+
+# NVM. Source: https://github.com/nvm-sh/nvm#installing-and-updating
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+```
+
+```shell script
+# Set up Aggie
+
+git clone https://github.com/TID-Lab/aggie.git
+cd aggie
+nvm install && npm install
+
+cp config/secrets.json.example config/secrets.json
+# User input: Customize Aggie settings per the README instructions.
+$EDITOR config/secrets.json
+
+# Ready! Test run:
+npm start
```

Comment:
The user should also start the `mongod` service for Aggie to successfully start up for testing:

```bash
sudo systemctl start mongod
npm start
sudo systemctl stop mongod
```